### ISSUE TITLE
Fix S0C5 in vsam1 demo program

### ISSUE THREAD

**Issue by dsh33782** [z390development/z390#253]
Using zvsam(1) with hybrid macros updated by Melvyn, now the first vsam1 demo program is abending with S0C5.
Here is log:

08:39:13 ESF1SP    EZ390 START USING z390 V1.7.07 ON J2SE 1.8.0_261 05/02/21
EZ390I Copyright 2011 Automated Software Tools Corporation
EZ390I z390 is licensed under GNU General Public License
EZ390I program = vsam1\demo\ESF1SP.390
EZ390I options = zvsam(1) zvsam(1)
ESF1SP CREATE ESF1.VES ESDS FIXED FILE FROM QSAM FILE
EZ390I Trace Table Entry      800FFD6A A715001F     BRAS 
EZ390I Trace Table Entry      800FFDA8 0A23         SVC  
EZ390I Trace Table Entry      800FFDAA 4110D1B0     LA   
EZ390I Trace Table Entry      800FFDAE 41000040     LA   
EZ390I Trace Table Entry      800FFDB2 0A13         SVC  
EZ390I Trace Table Entry      800FFDB4 12FF         LTR  
EZ390I Trace Table Entry      800FFDB6 4770D0CA     BNE  
EZ390I Trace Table Entry      800FFDBA 4110D218     LA   
EZ390I Trace Table Entry      800FFDBE 41000020     LA   
EZ390E error  11 ABEND PSW=07850600 800FFDC2 0A13         SVC   ABEND S0C5
 R0-R3 0000000000000000 F4F4F4F4000FF7C8 F4F4F4F4F4F4F4F4 F4F4F4F4F4F4F4F4
 R4-R7 F4F4F4F4F4F4F4F4 F4F4F4F4F4F4F4F4 F4F4F4F4F4F4F4F4 F4F4F4F4F4F4F4F4
 R8-RB F4F4F4F4F4F4F4F4 F4F4F4F4F4F4F4F4 F4F4F4F4F4F4F4F4 F4F4F4F4F4F4F4F4
 RC-RF F4F4F4F4F4F4F4F4 F4F4F4F4800FFD00 F4F4F4F400002018 F4F4F4F400000000
EZ390E error  12 program aborting due to abend S0C5
EZ390I instructions/sec     = 234
EZ390I total errors         = 1
08:39:13 ESF1SP    EZ390 ENDED   RC=16 SEC= 0 MEM(MB)= 22 IO=180 INS=15

**Comment by jyganci** [comment]
The S0C5 is due to a new variable added to sz390.java in RPI 1598. See z390_v1707.zip version of sz390.java, line 845:
```
LinkedList<zACB> zACB_list;                            // RPI 1598
```
The line should be
```
LinkedList<zACB> zACB_list = new LinkedList<zACB>();   // RPI 1598
```

**Comment by abekornelis** [comment]
@jyganci John, would you please implement the fix and create pull request?

**Comment by jyganci** [comment]
Don's June 20, 2021 email indicates he is working this issue.

**Comment by dsh33782** [comment]
This issue was corrected as part of issue #245
